SignalModelInterpolation/create_signal_ws_new_cat_2d.py

```diff
@@ -18,36 +18,18 @@ def unique(a, b):
   return 0.5*(a+b)*(a+b+1)+b
 
 def getNuisanceDatacardName(name, year):
-  if name == "fnufUnc":
-    return "CMS_hgg_nuisance_funf_13TeVscaleCorr"
-  elif name == "materialUnc":
+  if name == "fnuf":
+    return "CMS_hgg_nuisance_fnuf_13TeVscaleCorr"
+  elif name == "material":
     return "CMS_hgg_nuisance_material_13TeVscaleCorr"
-  elif name == "PhoSmearUnc":
+  elif name == "MCSmear_smear":
```

Comment:
I agree, however, this is a bit complicated and we also need to do the changes on XtoYH_pNN repository since it's a variable name from there, and also the pre-selection folder name.. so maybe we can keep this for simplicity, and change in later PRs if it's not confusing?